Formula/timew.rb

```diff
@@ -0,0 +1,18 @@
+class Timew < Formula
+  desc "Timewarrior is a command-line time tracking application"
+  homepage "https://taskwarrior.org/docs/timewarrior/"
+  url "http://taskwarrior.org/download/timew-1.0.0.tar.gz"
+  sha256 "ac027910e1e8365bdd218a8b42389b26d017d38d3c96516c408db6d5a44e0bb5"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "timew"
```

Comment:
Also, just calling the executable isn't a good test. It's best to ask it do something, then somehow validate the output.